content/git/concepts/restore/restore.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Restore'
+Description: 'The git restore command undoes changes in the working tree by restoring content from a source.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+The **`restore`** command is a versatile tool that enables developers to restore files to a specific state. This functionality allows you to undo any changes made to files in both your working directory and staging area. This command is especially valuable when you need to revert alterations, whether they have been staged for a commit or not.
+
+## Syntax
+
+The syntax of the `git restore` command is as follows:
+
+```shell
+git restore <options> -- <file>
+```
+
+## Example
+
+In this example, the initial [`git status`](https://www.codecademy.com/resources/docs/git/status) command shows that there are changes in the `index.html` file that have not been staged for a commit. It suggests using [`git add`](https://www.codecademy.com/resources/docs/git/add) to stage these changes. However, the user decides to use `git restore index.html` to discard the changes in the working directory and revert the file to its last committed state. After running this command and checking [`git status`](https://www.codecademy.com/resources/docs/git/status) again, the working directory is now clean, indicating that there are no pending changes, and there's nothing left to commit.
```

Comment:
[suggestion] From there, you can modify the description based on your new code snippet
```suggestion
In this example, the initial [`git status`](https://www.codecademy.com/resources/docs/git/status) command shows that there are changes in the `index.html` file that have not been staged for a commit. It suggests using [`git add`](https://www.codecademy.com/resources/docs/git/add) to stage these changes. However, the user decides to use `git restore index.html` to discard the changes in the working directory and revert the file to its last committed state. After running this command and checking [`git status`](https://www.codecademy.com/resources/docs/git/status) again, the working directory is now clean, indicating that there are no pending changes, and there's nothing left to commit.
```